src/app/core/services/account/account.service.ts

```diff
@@ -0,0 +1,73 @@
+import { Injectable } from '@angular/core';
+import {BehaviorSubject, concatMap, EMPTY, from, map, Observable, of} from "rxjs";
+import {ActivatedRoute, Router} from "@angular/router";
+import {HttpClient} from "@angular/common/http";
+
+interface Account {}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AccountService {
+  private accountSubject: BehaviorSubject<Account>;
+  public account: Observable<Account>;
+
+  constructor(
+      private router: Router,
+      private route: ActivatedRoute
+  ) {
+    this.accountSubject = new BehaviorSubject<any>({});
```

Comment:
Will there be a valid, initial value for the account? If so, leave it. If the first value is empty/null, use ReplaySubject(1)